pywr-schema/src/parameters/mod.rs

```diff
@@ -490,6 +513,7 @@ impl TryFromV1Parameter<ParameterV1> for Parameter {
                 }
                 CoreParameter::RollingMeanFlowNode(_) => todo!("Implement RollingMeanFlowNodeParameter"),
                 CoreParameter::ScenarioWrapper(_) => todo!("Implement ScenarioWrapperParameter"),
+                CoreParameter::WeeklyProfile(_) => todo!("Implement WeeklyProfileParameter"),
```

Comment:
The three new entries in this match (`WeeklyProfile`, `NegativeMax` and `NegativeMin`) all have existing entries that should be updated instead of these being added.

Reply:
This has been sorted